plotting/hovmoller.py

```diff
@@ -61,50 +59,24 @@ def find_depth(depth, clip_length, dataset):
             self.depth, self.depth_value, self.depth_unit = find_depth(
                 self.depth, len(dataset.depths) - 1, dataset)
 
-            time_var = dataset.nc_data.time_variable
-            if self.starttime > 0:
-                self.starttime = dataset.nc_data.timestamp_to_time_index(
-                    self.starttime)
-            if self.endtime > 0:
-                self.endtime = dataset.nc_data.timestamp_to_time_index(self.endtime)
-
-            time = time_var[slice(min(self.starttime, self.endtime), max(
-                self.starttime, self.endtime) + 1)].values
-
-            if len(self.variables) > 1:
```

Comment:
this if-block is dead code; a relic from when we didn't have the calculation layer to produce the magnitude of two components so we did it manually. There are tons of other spots in the code that do this, but I figured since I'm editing this file already, let's just start cleaning it up.